_til/git/intro-to-git.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Introduction to Git
+category: git
+---
```

Comment:
Make an introduction to Git as a version control system, what a version control system is, how git is different.